.github/workflows/deploy_infra.yml

```
@@ -0,0 +1,59 @@
+name: Deploy Infrastructure
+
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'
+        required: true
+        default: 'warning'
+        type: choice
+        options:
+        - info
+        - warning
+        - debug
+      environment:
+        description: 'The environment used as target'
+        type: choice
+        required: true
+        default: dev
+        options:
+        - dev
+        - prod
```

```suggestion
        default: prod
        options:
        - prod
```
just saw this. currently we do not have a dev environment

---

Correct